CONTRIBUTING.md

```diff
@@ -249,6 +249,17 @@ instead of adding a new import to the top of the file. If you are using a type a
 can add an import with a comment. However, if its only used a couple of times, the FQN is preferred to prevent future
 patch conflicts in the import section of the file.
 
+### Nullability annotations
+
+We are in the process of switching nullability annotation libraries, so you might need to use one or the other:
+
+**For classes we add**: Fields, method parameters and return types that are nullable should be marked via the
+`@Nullable` annotation from `org.jspecify.annotations`. Whenever you create a new class, add `NullMarked`, meaning types
+are assumed to be non-null by default.
```

Comment:
Maybe can add a reference for the JSpecify docs?
like: https://jspecify.dev/docs/user-guide/